astropy/visualization/tests/test_time.py

```diff
@@ -6,12 +6,18 @@
 pytest.importorskip('matplotlib')  # noqa
 
 import matplotlib.pyplot as plt
+import matplotlib.dates
 from erfa import ErfaWarning
 
 from astropy.time import Time
 from astropy.visualization.time import time_support
 from astropy.utils.compat.context import nullcontext
 
+# Matplotlib 3.3 added a settable epoch for plot dates and changed the default
+# from 0000-12-31 to 1970-01-01. This can be checked by the existence of
+# get_epoch() in matplotlib.dates.
+MPL_EPOCH_1970 = hasattr(matplotlib.dates, 'get_epoch')
```

Comment:
This assumes the user running the tests has not overridden this... I'm happy to go by that assumption but perhaps mention it in the comment above?

Reply:
Is that a configuration setting? Otherwise I don't know how a user could set that prior to running this test. Maybe I'm missing something obvious.

Reply:
I think it is indeed configurable - see https://matplotlib.org/3.3.1/users/whats_new.html#dates-use-a-modern-epoch, in particular
```
Access to the epoch is provided by get_epoch, and there is a new rcParams["date.epoch"] (default: '1970-01-01T00:00:00') rcParam.
```
I guess the alternative would be to actually call `get_epoch` as you do in the code proper...

Reply:
Because the test is about the string tick values, I'm not sure it is going to be possible or easy to get this right for an arbitrary user epoch. In any case there are many other battles to fight that are probably more worth the time, so I'll accept your offer to just merge now. :smile: